src/cms/views/authentication/authentication_actions.py

```diff
@@ -270,9 +271,31 @@ def password_reset_complete(request):
     logger.info("Password of %r was reset", request.user.profile)
     messages.success(
         request,
-        (
-            _("Your password has been successfully changed.")
-            / _("You can now log in with your new password.")
+        __(
+            _("Your password has been successfully changed."),
+            _("You can now log in with your new password."),
+        ),
+    )
+    return HttpResponseRedirect(reverse("login"))
+
+
+def account_activation_complete(request):
+    """
+    View to indicate that the account was successfully activated
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :return: Redirect to login page after activation
+    :rtype: ~django.http.HttpResponseRedirect
+    """
+
+    logger.info("Account activation for user %r was successful", request.user.profile)
```

Comment:
```suggestion
```
This will raise the error `'AnonymousUser' object has no attribute 'profile'` if the user is not authenticated while he is setting his password (which will almost always be the case). Also, there is no guarantee that the currently logged in user is the same user as the one being activated. So I'd propose to move this logging message into the `save()`-method of the `ActivationForm`.

Reply:
Interesting, I never saw this exception because I always was logged in as root, good point :+1: